project/standard/static/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM tomcat:9-jdk11 AS mother
+LABEL maintainer="Alessandro Parma<alessandro.parma@geosolutionsgroup.com>"
+ARG MAPSTORE_WEBAPP_SRC="https://github.com/geosolutions-it/MapStore2/releases/latest/download/mapstore.war"
```

Comment:
This will copy mapstore from latest release. This is not what we want for project, or it is unuseful and they can take directly the docker container from dockerhub.
By the way, if this is the standard docker file, a future build of the docker will build always against the altest version, and this is not what we want. So this looks to be a bug of the current dockerfile too.